packages/common-ui-web/src/components/VehiclePartSelection/VehiclePartSelection.tsx

```diff
@@ -0,0 +1,85 @@
+import { PartSelectionOrientation, VehiclePart, VehicleType } from '@monkvision/types';
+import { useEffect, useState } from 'react';
+import { useMonkTheme } from '@monkvision/common';
+import { Icon } from '../../icons';
+import { VehicleDynamicWireframe, VehicleDynamicWireframeProps } from '../VehicleDynamicWireframe';
+import { styles } from './VehiclePartSelection.style';
+
+export interface VehiclePartSelectionProps {
+  /**
+   * Vehicle type to display the wireframe for.
+   */
+  vehicleType: VehicleType;
+  /**
+   * The initial orientation of the wireframe when the component is mounted.
+   */
+  orientation?: PartSelectionOrientation;
+  /**
+   * Emit when change in the selected parts.
+   */
```

Comment:
`Callback called when the selected parts are updated (the user selects or unselects a new part).`